ddtrace/tracer/tracer.go

```diff
@@ -450,6 +453,33 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 		// if not already sampled or a brand new trace, sample it
 		t.sample(span)
 	}
+	var labels []string
+	if t.config.profilerHotspots {
+		labels = append(labels, "span id", fmt.Sprintf("%d", span.SpanID))
+	}
```

Comment:
Shouldn't this go into the `if` statement below? None of this is used outside of it.